File-Formats/PDF/Working-with-OCR/Docker.md

```diff
@@ -0,0 +1,124 @@
+---
+title: Performing OCR in Docker | Syncfusion
+description: Learn how to Perform OCR in Docker with easy steps using Syncfusion .NET OCR library.
```

Comment:
Learn how to perform OCR on scanned PDF documents and images in Docker with different tesseract version using Syncfusion .NET OCR library.

Reply:
Changed the content as given